### ISSUE TITLE
GitHub Enterprise support would be nice

### ISSUE THREAD

**Issue by jeoffw** [gitscout/gitscout-feedback#58]
I just tried it and the login appears to only support github.com.

**Comment by jeremybenaim** [comment]
@jeoffw we never had that case before. Will definitely check out whats needed to support Github Enterprise. Thanks for reporting this!

**Comment by jasonmccay** [comment]
Yep ... Github is quickly becoming the new standard within IBM (where I am). We rely on the Enterprise version of GH. Totally get your focus on public GH first ... will be looking forward to support for Enterprise.

**Comment by jeremybenaim** [comment]
Thanks @jasonmccay we'll try to fix this as soon as possible! There's no reason why Github Enterprise users would not be allowed to use Gitscout! 😉

**Comment by jeoffw** [comment]
@jasonmccay Yes, I was trying to use it to log in to IBM GitHub Enterprise : )

**Comment by tborychowski** [comment]
Hi, same here (SAP).
This app looks great and like a life saver, so cannot wait for this feature.

From my experience with GH API - this should be relatively simple to implement. The GH API is the same, so it's just a matter of allowing us to provide the GH base URL :-)

**Comment by jeremybenaim** [comment]
@jasonmccay @jeoffw @tborychowski hey guys, we just started working on the Enterprise  integration. But we figured it'd be better to work closely with you on this. Do you mind trying our nightly build when it'll be available?

If so, please sent us an email so that we can give you the link to download the nightly.

Thanks 😊

**Comment by jasonmccay** [comment]
Email sent.

**Comment by k207899** [comment]
Sorry, I didn't notice that before, but it seems that the app:
- is not open-source
- it requires an account and sends some data to your servers
- it includes some tracking 
So, unfortunately, from the privacy and data security reasons, I won't be able to participate in the beta tests.

**Comment by tborychowski** [comment]
Sorry, I didn't notice that before, but it seems that the app:
* is not open-source
* it requires an account and sends some data to your servers
* it includes some tracking So, unfortunately, from the privacy and data security reasons, I won't be able to participate in the beta tests.

**Comment by jeremybenaim** [comment]
@tborychowski we completely understand but hope you'll have another opportunity to use Gitscout (e.g. for personal projects, etc).

That being said, apart from being open source, which is unfortunately not our plan, I'd love to hear your inputs on what you think Gitscout would need to be used at SAP 🙂
For instance, an on-premise installation of our Gitscout API on your server – hence, no tracking – or something else?

Thanks 🙏

**Comment by tborychowski** [comment]
@jeremybenaim,
Yes, I believe that an on-premise installation would allow us to give it a go :-)

**Comment by jeremybenaim** [comment]
@tborychowski thanks! Be sure that we'll investigate on how to make this happen in a not so distant future and you'll be the first to know ;)

**Comment by smashedlife** [comment]
I would be more than willing to help test the Github Enterprise integration if you need more people. We also use it at my work.

**Comment by jeremybenaim** [comment]
hi @smashedlife thanks, we're working on it! Please send us an email at hello@gitscout.com so that we can reach out when a build is ready for you to try! 🙂

**Comment by the-reverend** [comment]
+1 at underarmour.com

**Comment by Jarvvski** [comment]
Another IBMer here, looking forward to GHE intergration
